package.json

```
@@ -4,37 +4,35 @@
   "description": "Application to connect Hemi Network to External Services",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "type": "module",
   "private": true,
   "scripts": {
-    "give-points": "bun ./src/presentation/cli/GivePoints.ts",
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "vitest run",
     "test:cov": "vitest run --coverage",
-    "test:watch": "vitest watch"
+    "test:watch": "vitest watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "collect-metrics": "node dist/presentation/cli/CollectMetrics.js",
+    "give-points": "node ./src/presentation/cli/GivePoints.ts"
   },
-  "author": "Thomas Alvarenga (thomas@bloq.com)",
+  "author": "Thomas Alvarenga (thomas@hemi.xyz)",
```

Using angles aligns to common practices.

```suggestion
  "author": "Thomas Alvarenga <thomas@hemi.xyz>",
```